requirements_orm.txt

```diff
@@ -1,3 +1,3 @@
-mysql-connector-python
+mysql-connector-python<=8.0.29
```

Comment:
Is there a specific need for this restriction?

Reply:
Yes, unfortunately it doesn't work with 8.0.30 - note the red cross on the previous commit. There might be some incompatibility with MariaDB or its default settings. See also https://github.com/ispyb/py-ispyb/pull/206 where they ran into the same/similar problem.